tests/php/PHPStanBearTest.py

```diff
@@ -0,0 +1,23 @@
+from bears.php.PHPStanBear import PHPStanBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+
+good_file = """
+<?php
+    $a = 1;
+?>
+"""
+
+
+bad_file = """
+<?php
+    $a =
+?>
+"""
+
+
+PHPStanBearTest = verify_local_bear(
```

Comment:
@damngamerz Sorry didn't knew that regarding `verify_local_bear` :disappointed: 
btw we should still use `check_results` over `check_validity` ( @Mixih please correct me if I'm wrong :sweat_smile: )